jest/hasteImpl.js

```diff
@@ -9,6 +9,7 @@
 
 'use strict';
 
+// TODO: remove this file entirely
```

Comment:
Can you remove the whole file in this update? Why not?

Reply:
Yep, trashed it!